### PR TITLE
Component for Empty Call To Action

### PR DESCRIPTION
**Pull request by spolu**
Make our empty call to actions a component to normalize visual across the app.

Screenshot from 2023-12-18 10-08-56

### CODE REVIEW

**Comment by spolu** [comment]
cc @Duncid

File: front/components/EmptyCallToAction.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  label: string;
+  disabled?: boolean;
+  icon?: ComponentType;
+  href?: string;
+  onClick?: (event: MouseEvent<HTMLButtonElement>) => void;
+}) {
+  const button = (
+    <Button
+      disabled={disabled}
+      size="md"
```
Comment (flvndvd) [inline front/components/EmptyCallToAction.tsx:23]: Should this be a prop?

File: front/components/EmptyCallToAction.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      label={label}
+      variant="primary"
+      icon={icon}
+      onClick={onClick}
+    />
+  );
+  return (
+    <div
+      className={classNames(
+        "relative flex h-full min-h-48 items-center justify-center rounded-lg bg-structure-50"
```
Comment (flvndvd) [inline front/components/EmptyCallToAction.tsx:33]:
```suggestion
        "flex h-full min-h-48 items-center justify-center rounded-lg bg-structure-50"
```

Comment (flvndvd) [reply front/components/EmptyCallToAction.tsx:33]: We could probably support `className` as a `prop` to allow overriding the behavior here.

File: front/components/EmptyCallToAction.tsx (hunk 1 above)
Comment (spolu) [reply front/components/EmptyCallToAction.tsx:23]: Right now we use the same everywhere so I think we can keep it a non prob until it is needed?

**Review by fontanierh — APPROVED** [review]

**Comment by Duncid** [comment]
LGTM, why not make it a Sparkle component?
Maybe part of <ContextItem>
<ContextItem.Empty> ?

**Review by Duncid — APPROVED** [review]

**Comment by spolu** [comment]
Removed relative
@Duncid yep that's step1 we can def move to sparkle as step 2

File: front/components/EmptyCallToAction.tsx (hunk 2 above)
Comment (spolu) [reply front/components/EmptyCallToAction.tsx]: Just removed the relative for now. Checked that it was not used :+1:

**Pull request merged**